src/main/webapp/app/pages/annotationPage/CancerTypeViewTable.tsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { SimpleTable } from 'app/components/SimpleTable';
+import { LEVELS } from 'app/config/constants';
+import { FdaLevelIcon, OncoKBLevelIcon } from 'app/shared/utils/Utils';
+import { CitationTooltip } from 'app/components/CitationTooltip';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { Citations } from 'app/shared/api/generated/OncoKbAPI';
+import { ImplicationDescriptionCell } from 'app/pages/annotationPage/ImplicationDescriptionCell';
+import { Else, If, Then } from 'react-if';
+import styles from './index.module.scss';
+
+export type DataType = 'tx' | 'dx' | 'px' | 'fda';
+
+export interface ITable {
+  userAuthenticated: boolean;
+  type: DataType;
+  hugoSymbol: string;
+  data: ITableRow[];
+}
+
+export interface ITableRow {
+  level: LEVELS;
+  alterations: string;
+  cancerTypes: string;
+  drugs?: string;
+  citations?: Citations;
+  description?: string;
+}
+
+const CompactDrugCell: React.FunctionComponent<{
+  hugoSymbol: string;
+  alterations: string;
+  cancerTypes: string;
+  drugs: string;
+}> = props => {
+  return (
+    <div style={{ minWidth: 300 }}>
+      <div>{props.drugs}</div>
+      <div className={'text-secondary'}>
+        Associated with {props.hugoSymbol} {props.alterations}
+      </div>
+      <div className={'text-secondary'}>
+        In {props.cancerTypes.toLowerCase()}
+      </div>
```

Comment:
Did we want to separate this into two lines or as a single sentence?

Reply:
They should be two lines if I remember correctly. But let's get some feedbacks tomorrow.